dipy/viz/window.py

```diff
@@ -417,46 +427,22 @@ def __init__(self, ren, title='DIPY', size=(300, 300),
             self.style = vtk.vtkInteractorStyleImage()
         elif self.interactor_style == 'trackball':
             self.style = vtk.vtkInteractorStyleTrackballCamera()
+        elif self.interactor_style == 'custom':
+            self.style = CustomInteractorStyle()
         else:
             self.style = interactor_style
 
         self.iren = vtk.vtkRenderWindowInteractor()
-        self.iren.SetRenderWindow(self.window)
```

Comment:
Okay this here will have to have a internal dialog. But it should be called from a button not from a keyword. In that way you have more freedom to type and edit labels on the fly.